src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -99,14 +97,8 @@ private void clearCachedInfo(String path) {
     }
 
     synchronized (this) {
-      pathToPreviouslyUsedOutline.put(path, outline);
-      // Visit the fields on the outline to get which calls are actual named tests.
-      if (!cachedCallToTestType.containsKey(path)) {
-        cachedCallToTestType.put(path, new HashMap<>());
-        populateTestTypeMap(outline, cachedCallToTestType.get(path), file);
-      }
-
-      return cachedCallToTestType.get(path);
+      cache.put(path, new OutlineCache(outline, file));
```

Comment:
nit: cache the OutlineCache entry as a local variable so you don't have to read it out of a cache.

Reply:
Done.